backend/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "example-app"
+version = "0.1.0"
+description = "Example dataspace application backend"
+authors = ["IOXIO"]
+packages = [{include = "app", from = "."}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
This does not match the README, if it doesn't depend on 3.10 then make sure the README is correct about what this *depends on*